pom.xml

```diff
@@ -41,6 +41,7 @@
         <module>infinispan-client-quickstart</module>
         <module>jms-quickstart</module>
         <module>kafka-quickstart</module>
+        <module>kafka-avro-schema-quickstart</module>
```

Comment:
What about having an apicurio and confluent version? So we would be able to reference the confluent version from the guide.

Reply:
👍🏻  on that but I'd rather do the second one separately, I want to check the rest client stuff first when the current guides and quickstarts are merged